tests/snmpsim/zynos.snmprec

```diff
@@ -64,66 +58,30 @@
 1.0.8802.1.1.2.1.3.7.1.3.228|4|92
 1.0.8802.1.1.2.1.3.7.1.3.229|4|93
 1.0.8802.1.1.2.1.3.7.1.3.230|4|94
-1.0.8802.1.1.2.1.3.7.1.3.301|4|129
-1.0.8802.1.1.2.1.3.7.1.3.302|4|130
-1.0.8802.1.1.2.1.3.7.1.3.303|4|131
-1.0.8802.1.1.2.1.3.7.1.3.304|4|132
-1.0.8802.1.1.2.1.3.7.1.3.305|4|133
-1.0.8802.1.1.2.1.3.7.1.3.306|4|134
-1.0.8802.1.1.2.1.3.7.1.3.307|4|135
-1.0.8802.1.1.2.1.3.7.1.3.308|4|136
-1.0.8802.1.1.2.1.3.7.1.3.309|4|137
-1.0.8802.1.1.2.1.3.7.1.3.310|4|138
-1.0.8802.1.1.2.1.3.7.1.3.311|4|139
-1.0.8802.1.1.2.1.3.7.1.3.312|4|140
-1.0.8802.1.1.2.1.3.7.1.3.313|4|141
-1.0.8802.1.1.2.1.3.7.1.3.314|4|142
-1.0.8802.1.1.2.1.3.7.1.3.315|4|143
-1.0.8802.1.1.2.1.3.7.1.3.316|4|144
-1.0.8802.1.1.2.1.3.7.1.3.317|4|145
-1.0.8802.1.1.2.1.3.7.1.3.318|4|146
-1.0.8802.1.1.2.1.3.7.1.3.319|4|147
-1.0.8802.1.1.2.1.3.7.1.3.320|4|148
-1.0.8802.1.1.2.1.3.7.1.3.321|4|149
-1.0.8802.1.1.2.1.3.7.1.3.322|4|150
-1.0.8802.1.1.2.1.3.7.1.3.323|4|151
-1.0.8802.1.1.2.1.3.7.1.3.324|4|152
-1.0.8802.1.1.2.1.3.7.1.3.325|4|153
-1.0.8802.1.1.2.1.3.7.1.3.326|4|154
-1.0.8802.1.1.2.1.3.7.1.3.327|4|155
-1.0.8802.1.1.2.1.3.7.1.3.328|4|156
-1.0.8802.1.1.2.1.3.7.1.3.329|4|157
-1.0.8802.1.1.2.1.3.7.1.3.330|4|158
-1.0.8802.1.1.2.1.3.7.1.3.401|4|193
-1.0.8802.1.1.2.1.3.7.1.3.402|4|194
-1.0.8802.1.1.2.1.3.7.1.3.403|4|195
-1.0.8802.1.1.2.1.3.7.1.3.404|4|196
-1.0.8802.1.1.2.1.3.7.1.3.405|4|197
-1.0.8802.1.1.2.1.3.7.1.3.406|4|198
-1.0.8802.1.1.2.1.3.7.1.3.407|4|199
-1.0.8802.1.1.2.1.3.7.1.3.408|4|200
-1.0.8802.1.1.2.1.3.7.1.3.409|4|201
-1.0.8802.1.1.2.1.3.7.1.3.410|4|202
-1.0.8802.1.1.2.1.3.7.1.3.411|4|203
-1.0.8802.1.1.2.1.3.7.1.3.412|4|204
-1.0.8802.1.1.2.1.3.7.1.3.413|4|205
-1.0.8802.1.1.2.1.3.7.1.3.414|4|206
-1.0.8802.1.1.2.1.3.7.1.3.415|4|207
-1.0.8802.1.1.2.1.3.7.1.3.416|4|208
-1.0.8802.1.1.2.1.3.7.1.3.417|4|209
-1.0.8802.1.1.2.1.3.7.1.3.418|4|210
-1.0.8802.1.1.2.1.3.7.1.3.419|4|211
-1.0.8802.1.1.2.1.3.7.1.3.420|4|212
-1.0.8802.1.1.2.1.3.7.1.3.421|4|213
-1.0.8802.1.1.2.1.3.7.1.3.422|4|214
-1.0.8802.1.1.2.1.3.7.1.3.423|4|215
-1.0.8802.1.1.2.1.3.7.1.3.424|4|216
-1.0.8802.1.1.2.1.3.7.1.3.425|4|217
-1.0.8802.1.1.2.1.3.7.1.3.426|4|218
-1.0.8802.1.1.2.1.3.7.1.3.427|4|219
-1.0.8802.1.1.2.1.3.7.1.3.428|4|220
-1.0.8802.1.1.2.1.3.7.1.3.429|4|221
-1.0.8802.1.1.2.1.3.7.1.3.430|4|222
+1.0.8802.1.1.2.1.4.1.1.4.120.128.1408|2|4
```

Comment:
You are not supposed to change an existing snmprec file. Only create new ones. And (re) calculate the JSON files. 
SNMPREC is a capture of the device, and JSON is the calculated as a "reference" on how LibreNMS sees the device.
And regression test will be compared again at every change of the source code  to detect any issue.

Reply:
I have reverted my zynos.snmprec changes. Can you just confirm that what I did to the JSON files was correct, as I ran a new test and renamed it to be zynos.json not zynos_xgs4600.json? It would not let me run a test without a variant. If this is wrong and you have another script you want me to run just let me know.